doc/main.html

```diff
@@ -610,15 +610,26 @@ <h4>Additional elements section</h4>
         <ul>
           <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>;</li>
           <li>a <code>title</code> attribute that provides a short description of the element;</li>
-          <li>a <code>href</code> attribute that links to the element itself, typically in the <code>doc/elements</code>
-          directory.</li>
+          <li>a <code>href</code> attribute that links to the element, as detailed below.</li>
         </ul>
 
+        <p>
+          Each additional element shall be managed in one of the following ways:
+        </p>
+        <ul>
+          <li>as a file in the doc/elements directory, in which case the link to the element shall be relative path to the file, e.g. <code>.elements/element.txt</code></li>
+          <li>in a separate SMPTE repository, in which case the link to the element shall be a versioned link to the repo, e.g. <code>https://github.com/SMPTE/html-pub/tree/631ed7a0312fa183c356a7b742129ce46f92736f</code></li>
+        </ul>
+
+        <p class="note">
+          It is advised to only referenced element repos managed by SMPTE to fullfil license requirements, see <a href="#bib-smpte-gh-om"></a> for more details. If an element has been created outside of SMPTE, it should be considered as a bibliographic reference instead. 
```

Comment:
```suggestion
          Information that is not an integral part of the document is cited using a normative or bibliographic reference. 
```

Reply:
adressed in https://github.com/SMPTE/html-pub/pull/137/commits/f8b7af5b54ab6fa653cca071cecbd16e482084c1

Reply:
Better in https://github.com/SMPTE/html-pub/pull/137/commits/73f2f5cfd27399c64489aca14a61682c38361991